Assets/_BForBoss/_Core/Scripts/EnergySystem/EnergySystemBehaviour.cs

```diff
@@ -42,8 +42,8 @@ private EnergyData EnergyData
 
         private EnergySystemConfigurationData _energySystemConfigurationData;
 
-        private readonly Queue<(EnergyAccruementType, float)> _accruedEnergyTypeQueue = new();
-        private readonly Queue<(EnergyExpenseType, float)> _expendEnergyTypeQueue = new();
+        private readonly Queue<(EnergyAccruementType, float)> _accruedEnergyTypeQueue = new Queue<(EnergyAccruementType, float)>();
```

Comment:
So my Rider kept bugging me about this, as it keeps telling me that this formatting is an error. So just made it explicit to make Rider happy.

Can revert if you want @m4rtint !
![Personal Gripe](https://github.com/PerigonGames/BForBoss/assets/23490604/fd9e62f6-bdbf-4501-b720-b83f55b628b8)

Reply:
It's realated to setting ur C# versoin compiler. either way works.